File: BoardGames.Domain/DataModels/GamesStore/TyrantsOfTheUnderdark/TyrantsOfTheUnderdarkSaveModel.cs
namespace BoardGames.Domain.DataModels.GamesStore.TyrantsOfTheUnderdark;

public class TyrantsOfTheUnderdarkSaveModel : TyrantsOfTheUnderdarkMatchModel
{
    public Guid MatchId { get; set; }
    
    public int MatchNumber { get; set; }

    public DateTime DateMatch { get; set; }

    public int TotalCount { get; set; }
}

File: src/app/dataModels/gamesStore/tyrantsOfTheUnderdark/tyrantsOfTheUnderdarkSaveModel.ts
import { TyrantsOfTheUnderdarkMatchModel } from "./tyrantsOfTheUnderdarkMatchModel";

export interface TyrantsOfTheUnderdarkSaveModel extends TyrantsOfTheUnderdarkMatchModel {
    matchId: string;
    matchNumber: number;
    dateMatch: Date;
    totalCount: number;
}
